docs/application/dotnet/guides/nui/text.md

```diff
@@ -11,7 +11,7 @@ There are three Text components to display and edit text on the screen:
 <a name="textLabel"></a>
 ## TextLabel
 
-The `TextLabel` is a class that displays a short text string. `Tizen.NUI.BaseComponents` namespace contains the class.
+The [TextLabel](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.TextLabel.html) is a class that displays a short text string. `Tizen.NUI.BaseComponents` namespace contains the class.

```

Comment:
Since TextLabel is a noun here, we can remove the article "The"

[TextLabel](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.TextLabel.html) is a class that displays a short text string. The `Tizen.NUI.BaseComponents` namespace contains the class.